components/tally/sources/common/common.mjs

```diff
@@ -0,0 +1,47 @@
+import tally from "../../tally.app.mjs";
+
+export default {
+  props: {
+    tally,
+    formId: {
+      propDefinition: [
+        tally,
+        "forms",
```

Comment:
I think it makes sense to be `form` because although the options will be a list of forms,  the user selects one `form`, then the prop value will be a `formId`. Semantically it makes more sense if it's in the singular form